app/controllers/base_media_controller.rb

```diff
@@ -19,4 +19,10 @@ def proxy_to_s3_via_nginx(asset)
 
     head :ok, content_type: asset.content_type
   end
+
+  def redirected_to_draft_assets_host_for?(asset)
```

Comment:
I find this method a bit odd but I suggest we go with it and change it later if/when we revisit this code.

Reply:
Yeah. I'm not super happy with it either. The method needs to return a value so the controller sub-class can decide whether to `return` or not, otherwise we end up with more duplication in the controller sub-classes. And passing the `asset` in as a parameter matches how `#proxy_to_s3_via_nginx` works. With both those in mind I found it hard to come up with a good name for the method.

Reply:
@chrisroos I've addressed your comment in a new PR: https://github.com/alphagov/asset-manager/pull/452